source/common/config/delta_subscription_state.cc

```diff
@@ -44,10 +44,16 @@ void DeltaSubscriptionState::updateSubscriptionInterest(const std::set<std::stri
   }
 }
 
+// Appends aliases passed in updates_to_these_aliases parameter to the aliases that will be sent in
+// the next update request to the management server.
```

Comment:
This really confuses me, why are aliases being sent?

Reply:
Deleted requested_aliases_ field and all methods associated with it.